dataservices/dataservices_test.go

```diff
@@ -0,0 +1,309 @@
+package main_test
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strings"
+
+	. "github.com/tidepool-org/platform/dataservices"
+	"github.com/tidepool-org/platform/service"
+	"github.com/tidepool-org/platform/user"
+	"github.com/tidepool-org/platform/version"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("The Dataservices client", func() {
```

Comment:
We need to figure a good way to do full integration testing including the route and middleware.  Will likely need to mock the user/permissions client.